src/getting-started/README.md

```diff
@@ -6,34 +6,28 @@ sidebar_label: Introduction
 
 ## Introduction
 
-Before you begin, read [Personium Architecture](../user_guide/001_Personium_Architecture.md).
+This document is an introductory article for those who are new to Personium. The purpose of this document is to familiarize you with the basics of Personium so that you can use it.
```

Comment:
This article introduces the basics of using Personium. Upon completing this tutorial, even first-timers will understand how to use Personium.

Reply:
Will apply it to the next pull request.